velodyne_driver/src/driver/CMakeLists.txt

```diff
@@ -1,26 +1,59 @@
+cmake_minimum_required(VERSION 3.5.0)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+set(CMAKE_CXX_STANDARD 14)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rmw_implementation REQUIRED)
+find_package(rcl REQUIRED)
+find_package(rclcpp REQUIRED)
+#find_package(rclcpp_components REQUIRED)
+find_package(rcutils REQUIRED)
+find_package(diagnostic_updater REQUIRED)
+find_package(tf2 REQUIRED)
+find_package(diagnostic_msgs REQUIRED)
+find_package(velodyne_msgs REQUIRED)
+find_package(gtest REQUIRED)
+find_package(console_bridge REQUIRED)
+
+# libpcap provides no pkg-config or find_package module:
+set(libpcap_LIBRARIES -lpcap)
+
+include_directories(include
+                    ${rclcpp_INCLUDE_DIRS}
+                    ${console_bridge_INCLUDE_DIRS}
+                    ${diagnostic_updater_INCLUDE_DIRS}
+                    ${rmw_implementation_INCLUDE_DIRS}
+                    ${velodyne_msgs_INCLUDE_DIRS}
+                    ${diagnostic_msgs_INCLUDE_DIRS}
+)
+#${rclcpp_components_INCLUDE_DIRS}
+
```

Comment:
Because of the `include_subdirectory()` directive in the top-level CMakeLists.txt, the items above this aren't necessary (they're already included in the top-level one). The `include_subdirectory()` in the top-level CMakeLists.txt tells CMake to add the CMakeLists.txt found in that subdirectory to the top-level one so they're processed as one file.